h/browser/chrome/lib/sidebar-injector.js

```diff
@@ -103,39 +109,14 @@
       });
     }
 
-    function injectIntoLocalDocument(tab) {
-      if (isPDFURL(tab.url)) {
-        return injectIntoLocalPDF(tab);
-      } else {
-        return Promise.reject(new h.LocalFileError('Local non-PDF files are not supported'));
-      }
-    }
-
-    function injectIntoRemoteDocument(tab) {
-      return isPDFURL(tab.url) ? injectIntoPDF(tab) : injectIntoHTML(tab);
-    }
-
-    function injectIntoPDF(tab) {
-      return new Promise(function (resolve, reject) {
-        if (!isPDFViewerURL(tab.url)) {
-          chromeTabs.update(tab.id, {url: getPDFViewerURL(tab.url)}, function () {
-            resolve();
-          });
-        } else {
-          resolve();
-        }
-      });
+     function injectIntoLocalDocument(tab) {
+      return Promise.reject(new h.LocalFileError('Local non-PDF files are not supported'));
     }
 
-    function injectIntoLocalPDF(tab) {
-      return new Promise(function (resolve, reject) {
-        isAllowedFileSchemeAccess(function (isAllowed) {
-          if (isAllowed) {
-            resolve(injectIntoPDF(tab));
-          } else {
-            reject(new h.NoFileAccessError('Local file scheme access denied'));
-          }
-        });
+    function reloadPDF(tab) {
```

Comment:
Not sure I understand why this is necessary. Doesn't it "take over" the first time?